specification/imds/data-plane/Microsoft.InstanceMetadataService/stable/2019-03-11/imds.json

```diff
@@ -605,7 +605,7 @@
       "in": "query",
       "type": "string",
       "enum": [
-        "2018-10-01"
+        "2018-03-11"
```

Comment:
The folder says 2019-03-11, which is correct?

Reply:
```suggestion
        "2019-03-11"
```